addons/sourcemod/scripting/szf/stocks.sp

```diff
@@ -742,6 +759,19 @@ stock void CheckClientWeapons(int iClient)
 	}
 }
 
+stock int GetOriginalItemDefIndex(int iIndex)
+{
+	int iOrigIndex;
+	
+	char sIndex[8];
```

Comment:
12 char length is min needed to fit for 32-bit int, though we'll never get that high for item def index